examples/asr/emformer_rnnt/README.md

```diff
@@ -0,0 +1,63 @@
+# Emformer RNN-T ASR Example
+
+This directory contains sample implementations of training and evaluation pipelines for an Emformer RNN-T streaming ASR model.
+
+## Usage
+
+### Training
+
+[`train.py`](./train.py) trains an Emformer RNN-T model using PyTorch Lightning. Note that the script expects users to have access to GPU nodes for training and provide paths to datasets and the SentencePiece model to be used to encode targets. The script also expects a file (--global_stats_path) that contains training set feature statistics; this file can be generated via [`global_stats.py`](./global_stats.py).
+
+### Evaluation
+
+[`eval.py`](./eval.py) evaluates a trained Emformer RNN-T model on a given dataset.
+
+## Model Types
+
+Currently, we have training recipes for the LibriSpeech and TED-LIUM Release 3 datasets.
```

Comment:
Could you have a brief description on what’s the difference between these models? I know vocab size is one. Is there any other?

Reply:
yeah, that's the only difference in the model architecture. the recipes themselves have some differences that are specific to the datasets — this should be apparent in the lightning modules

Reply:
Sure, will do.